eth/config.go

```diff
@@ -181,6 +182,9 @@ type Config struct {
 	// send-transction variants. The unit is ether.
 	RPCTxFeeCap float64 `toml:",omitempty"`
 
+	// Verbosity
+	Verbosity int `toml:",omitempty"`
```

Comment:
This is very out of scope here. Log verbosity is relevant for Geth as a binary, it is not relevant for an internal subsystem.

Reply:
Sorry, but I didn't understand. The whole point of the requested feature (#21492) should be to move verbosity setting from an `interal` parameter that you can set only via command line to a config parameter that you can set via TOML config file. `Config` is the struct geth uses to marshal/unmarshal object to/from TOML file, so `Verbosity` must be there as other parameters. No?

Reply:
I think what @karalabe meant here is that it should not be in eth.Config, it should be in the gethConfig.